core/code/map_data_render.js

```diff
@@ -259,7 +261,7 @@ window.Render.prototype.createPlaceholderPortalEntity = function(guid,latE6,lngE
   if (guid in window.portals) {
     var p = window.portals[guid];
     portalMoved = latE6 !== p.options.data.latE6 || lngE6 !== p.options.data.lngE6;
-    if (team !== p.options.data.team) {
+    if (team !== p.options.data.team && p.options.timestamp < timestamp) {
```

Comment:
It looks working, but now It looks redundant with parts of the `createPortalEntity` function. Is it possible to check this at one place ?

Reply:
I don't think so. Portal placeholders still have 0 timestamp and we check here the actual timestamp.
Placeholders with proper timestamp made portals disappear much too often (with incoming link).

So we check here the actual timestamp to delete the entity only when faction changed, the other check then works on proper portal data and only checks this timestamp for entity data changes/updates

Reply:
oh right, I though the placeholder gets the link/field timestamp.